plaidml/core/core.h

```diff
@@ -47,7 +47,14 @@ T call(edsl_source_location loc, F fn, Args... args) {
   auto ret = fn(&err, args...);
   if (err.code) {
     std::stringstream ss;
-    ss << "Exception at " << loc.file_name() << ":" << std::to_string(loc.line()) << " with message: " << str(err.msg);
+    if (!strstr(loc.file_name(), "edsl/edsl.h")) {
```

Comment:
Avoid using `strstr` for c++ code, you can use `std::string::find` as an alternative.